wpsc-includes/variations.class.php

```diff
@@ -296,3 +296,30 @@ function wpsc_get_child_object_in_terms_var( $parent_id, $terms, $taxonomies, $a
 		return false;
 	}
 }
+
+/**
+ * Given array of variation selections this works through the terms and returns the product_id for the matching variation
+ *
+ * @since 4.0
+ * @author Curtis McHale
+ *
+ * @param array     $variations     required            The array of variation selections
+ * @uses wpsc_get_child_objects_in_terms()              Given $product_id and product params this returns the variation product id
+ * @return int      $product_id                         The ID of the variation product
+ */
+function wpsc_get_product_id_from_variation_selections( $variations ){
```

Comment:
Let's call this `wpsc_get_product_id_from_variations`.  

Also, need a space between the bracket and closing parentheses.